### ISSUE TITLE
Dynamic registration decal not work.

### ISSUE THREAD

**Issue by arv187** [flybywiresim/a32nx#7438]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2022-08-26T09:51:11+00:00",
    "ref": "refs/heads/master",
    "sha": "e46e611d618036e7bca32c5652f4806834665d93",
    "actor": "frankkopp",
    "event_name": "manual",
    "pretty_release_name": "master:e46e611d"
}
```

### Describe the bug

I can't see now the tail number, before it did, I have set in customization airplane -> tail number: EC-ZAB and in flypad dynamic registration decal On.

### Expected behavior

Tail Number is showed in exterior aircraft in both the default library and my custom library, both of which worked correctly before.

### Steps to reproduce

1. Empty community folder except a32nx from installer/launcher
2. Launch sim, go to world
3. Select a32nx and select fbw livery
4. set in customization airplane -> tail number: EC-ZAB
5. In flypad dynamic registration decal On
6. Fly
7. Exterior camera

### References (optional)

![imagen](https://user-images.githubusercontent.com/16170732/186910221-0d03717c-15d1-4f48-88d6-aae8b011c403.png)
![unknown](https://user-images.githubusercontent.com/16170732/186910648-0d17995e-3e50-49cf-b947-6887949982d6.png)
![unknown2](https://user-images.githubusercontent.com/16170732/186910653-15c984f8-2a14-4a00-8869-f586a2fd72fa.png)
![unknown3](https://user-images.githubusercontent.com/16170732/186910659-4246443d-2127-4347-b495-0179a205c565.png)

### Here you can see how previously this worked:
![imagen](https://user-images.githubusercontent.com/16170732/186913280-9e19c6e2-af91-4243-a4a4-3cec91965c50.png)

### Additional info (optional)

I haven't flown this aircraft for about 5 months and I didn't notice this fault until today. Previously it was working. Other planes, both with default textures and with other textures I have created, work.
![imagen](https://user-images.githubusercontent.com/16170732/186913647-b16578d7-ab00-47e7-81ba-89ec48d55557.png)

### Discord Username (optional)

Spark_23#2821

**Comment by tracernz** [comment]
Thanks for the report! This has been broken since flyPadOS v3 merged.

**Comment by arv187** [comment]
This morning I uninstalled again the development version, and installed the stable version 0.8.2. It was still not working on all fbw liveries, included default. In all others airplanes were working correctly.

I don't know how finally this work. I activated the dev mode, after that seeing display info about simobjecs, statistics profile, finally I activated the wireframe mode, and then the registration decal was there. Likewise, I don't know how.

I disabled the dev mode and still working in all fbw liveries.
Just now I uninstalled stable version to install the development, all working, registration decal are again visible.

Why? It's in sim something cached?
